File: 02_ProgrammingFundamentals/_10_TextProcessing/_01_Lecture/_02_RepeatStrings.java
package _02_ProgrammingFundamentals._10_TextProcessing._01_Lecture;

import java.util.Scanner;

public class _02_RepeatStrings {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        String[] console = scan.nextLine().split(" ");

        // Repeat string
        for (String word : console) {
            String repeatedWord = repeatWord(word, word.length());
            
            // Print result
            System.out.print(repeatedWord);
        }

        scan.close();
    }

    // Method that concatenates a string multiple times
    private static String repeatWord (String word, int length) {
        StringBuilder result = new StringBuilder();
        
        for (int i = 0; i < length; i++) {
            result.append(word);
        }
        
        return result.toString();
    }
}

File: 24_C#Fundamentals/_08_TextProcessing/_01_Lecture/_02_RepeatStrings/_02_RepeatStrings.cs
using System;
using System.Text;

public class _02_RepeatStrings
{
    public static void Main(string[] args)
    {
        // Read input values
        string[] console = Console.ReadLine().Split(" ");

        // Repeat string
        foreach (string word in console)
        {
            string repeatedWord = repeatWord(word, word.Count());

            // Print result
            Console.Write(repeatedWord);
        }
    }

    // Method that concatenates a string multiple times
    private static string repeatWord(string word, int times)
    {
        var result = new StringBuilder();
        for (int i = 0; i < times; i++)
        {
            result.Append(word);
        }

        return result.ToString();
    }
}
